scripts/cluster_k3d.sh

```diff
@@ -23,6 +23,7 @@ if [ "${1:-}" == 'create' ]; then
         --image rancher/k3s:$K3S \
         -s $MASTER_COUNT -a $WORKER_COUNT \
         --registry-create k3d-$CLUSTER_NAME-registry \
+        --registry-config <(echo "${K3D_REGISTRY_CONFIG:-}") \
```

Comment:
LGTM. But I have a question. Will it fail if I do not have the environment variable set?

Reply:
I share this concern, maybe it's better to not pass the flag if the var is not set.

Reply:
I tried running locally without variable and it works fine.
Bash variable expansion `${var:-}` produces empty string - so we get `--registry-config <(echo "")` - which means use empty config file.